doc/cephfs/quota.rst

```diff
@@ -1,10 +1,52 @@
-Quotas
-======
+CephFS Quotas
+=============
 
-CephFS allows quotas to be set on any directory in the system.  The
+CephFS allows quotas to be set on any directory in the file system.  The
 quota can restrict the number of *bytes* or the number of *files*
 stored beneath that point in the directory hierarchy.
 
+Like most other things in CephFS, quotas are configured using virtual
+extended attributes:
+
+ * ``ceph.quota.max_files`` -- file limit
+ * ``ceph.quota.max_bytes`` -- byte limit
+
+If the extended attributes appear on a directory that means a quota is
+configured there. If they are not present then no quota is set on that
+directory (although one may still be configured on a parent directory).
+
+To set a quota, set the extended attribute on a CephFS directory with a
+value::
+
+  setfattr -n ceph.quota.max_bytes -v 100000000 /some/dir     # 100 MB
```

Comment:
Once https://github.com/ceph/ceph/pull/49008 is merged, the quota value can be human readable.

Reply:
Yes, the corresponding doc change has been made on that PR.